src/Symfony/Component/ErrorHandler/ErrorRenderer/ErrorRenderer.php

```diff
@@ -62,7 +62,7 @@ public function render($exception, string $format = 'html'): string
             throw new ErrorRendererNotFoundException(sprintf('No error renderer found for format "%s".', $format));
         }
 
-        if (!$exception instanceof FlattenException) {
+        if ($exception instanceof \Exception) {
```

Comment:
Shouldn't we check for Traversable here ? I think we need to let the component render `Error` too

Reply:
You mean `Throwable`?

Reply:
yeah, indeed.

Reply:
Done.